projects/igniteui-angular/src/lib/directives/toggle/toggle.directive.ts

```diff
@@ -173,6 +173,10 @@ export class IgxToggleDirective implements IToggleView, OnInit, OnDestroy {
      * ```
      */
     public open(overlaySettings?: OverlaySettings) {
+        if (!this._collapsed) {
```

Comment:
OK, question: isn't this required for the tooltip to actually replay its animation, and if so, wouldn't this break the tooltip behavior?

Reply:
igxTooltip should work correctly now